src/ontology/mondo-edit.obo

```diff
@@ -355678,6 +355683,7 @@ synonym: "chromosome 20 ring" RELATED [GARD:0001334]
 synonym: "R20" RELATED ABBREVIATION [GARD:0001334]
 synonym: "Ring 20" RELATED [GARD:0001334]
 synonym: "Ring chromosome 20 syndrome" RELATED [GARD:0001334]
+synonym: "ring chromosome 20 syndrome" EXACT [PMID:37498137]
```

Comment:
@nicolevasilevsky the other "ring chromosome 20 syndrome" x-refs are related. Could you please update according to whether the synonym should be exact or related? Thank you!

Reply:
fixed